src/Arcus.Template.WebApi/Startup.cs

```diff
@@ -39,6 +42,20 @@ public void ConfigureServices(IServiceCollection services)
             });
 
             services.AddHealthChecks();
+
+//#if DEBUG
+            var openApiInformation = new Info
+            {
+                Title = "Arcus.Template.WebApi",
```

Comment:
Can we plug in the name of the project that was created based on this template?

Reply:
Yep, all occurences of the project name (Arcus.Template.WebApi) will be replaced by the name you give the created project.

(Copy-Pasta 🙈)

Reply:
![image](https://user-images.githubusercontent.com/9039753/66037659-6ea42780-e510-11e9-8e3d-cf3aac6be2b8.png)

For example.

Reply:
Awesome, thanks!